### DISCUSSION TITLE
Meeting notes #3, 2022-10-04

### DISCUSSION THREAD

**Discussion by jongfeel** [wisdom08/dev-community#29] [General]
### 로그인 작업 진행에 대한 회고

- 뭔가 찾아서 시간을 보내는 건 맞는데 결과가 안 나오고 있음

=> 대처 방법
- 진행한 내용을 공유를 잘 해보자, 잘 되든 안 되든
- 질문을 하자, 잘 안되면 내용 공유를 한 뒤 피드백을 기다려 보자

### 스스로에게 피드백을 줘야 함

- 구체적인 작업 계획을 적어 놓으면 진행하면서도 내용이 변경이 일어남
- 과거의 자신의 생각과 현재 변경된 자신의 생각을 업데이트 하면서 작업의 진행 결과를 판단할 수 있음
- 꼭 코드가 아니더라도 가능

### 작업 완료 기준을 정하자

- 어떤 작업이 진척이 안되는 건, 완료 기준이 없기 떄문일 수 있다.
- 작업의 완료 기준이 없다면 작업이 큰 것이고 완료 기준이 있어도 더 세분화해서 작업을 구분해 볼 수 있음

```
게시판 CRUD에서 C의 완료 기준의 예)

1. 게시글 작성 버튼을 누른다
2. 제목과 내용을 적는다
3. 제출 버튼을 누른다
4. 게시판 목록에서 방금 작성한 글을 확인한다.
```

이정도 당연한 내용은 코딩을 안 해도 적을 수 있습니다.

### 자기 자신의 능력치를 잘 알아야 함

- Iteration 2도 '설마 이걸 못할까?'로 생각하고 범위를 잡았지만, 그것이 실제로 일어났음.
- 자신의 시간 투자 대비 결과물이 자신의 실력이라고 인정해야 함.
- 계획한 작업 단위 만큼 코드를 작성하고 그 작업의 내용과 코드가 일치할 때 까지 훈련이 필요

### Iteration 3 작업 시간

- 면접 준비로 주말 오전 3시간만 쓸 예정
- 3시간 작업 분량을 잡고 구체적인 작업을 적어보고 진행
- 다음 미팅은 짧게 얘기하는 것으로

**Comment by wisdom08** [comment]
멘토님 작업 완료 기준을 정하는 걸 최우선으로 진행해보겠습니다. 
감사합니다!
